client-datastore-cassandra/src/test/java/com/telecominfraproject/wlan/client/datastore/cassandra/ClientDatastoreCassandraTests.java

```diff
@@ -54,12 +54,12 @@
     
     @Override
     protected List<String> getClientSessionPagination_expectedPage1EmptySortStrings(){
-        return new ArrayList<>(Arrays.asList(new String[]{"qr_30", "qr_43", "qr_18", "qr_9", "qr_10", "qr_23", "qr_20", "qr_47", "qr_1", "qr_40" }));
+        return new ArrayList<>(Arrays.asList(new String[]{"qr_18", "qr_34", "qr_0", "qr_27", "qr_16", "qr_24", "qr_31", "qr_4", "qr_30", "qr_49" }));
```

Comment:
These values were taken from running the unit tests against a local Cassandra deployment after clearing the data and running from clean tables